docs: Create initial biolatency guide

blanquicet opened the pull request:
This PR implements a simple guide about the biolatency gadget. It could be still improved by using additional flags of the `stress` tool and when we will have the possibility to use the different options that the BCC biolatency script offers.

Fixes #309

alban commented on docs/guides/biolatency.md:
```suggestion
# Run stress with 1 worker that will generate I/Os
```

alban commented on docs/guides/biolatency.md:
```suggestion
The new histogram shows how the number of I/Os increased significantly,
```

alban reviewed (commented):
> Some minor comments. I'm not sure about the spelling of I/O's. Should it be just 'I/O'?

Or I/Os?

blanquicet commented on docs/guides/biolatency.md:
Not necessary, I just wanted to increase the samples and reduce the possibility of having outliers.

mauriciovasquezbernal replied:
Can we reduce it? IMO it'll be very annoying to have to wait for 3 minutes twice while doing this PR. Perhaps 1 minute is enough and we can put a comment indicating a longer wait would produce more stable results?

blanquicet replied:
Done.

blanquicet commented:
> > Some minor comments. I'm not sure about the spelling of I/O's. Should it be just 'I/O'?
>>
> Or I/Os?
>

@mauriciovasquezbernal @alban I modified it to "I/O operations" or simply "I/O" where "operations" can be omitted.

In addition, I also added the reference to the BCC documentation.

mauriciovasquezbernal commented on docs/guides/biolatency.md:
````diff
@@ -0,0 +1,130 @@
+```
+
+This output shows that the bulk of the I/O was between 64 and 1023 us, and
+that there were 1544 I/O operations during the time the gadget was running.
````
did you sum it up by hand? :thinking:

blanquicet replied:
Unfortunately yes, that's something the tool provides using the option `-e`, see [here](https://github.com/iovisor/bcc/blob/67f59ee80fcf5deedaacba1436d9fa09d32a16a0/tools/biolatency_example.txt#L296-L314) but we don't have the possibility of using flags yet. 

I think we should definitely extend the trace CR to allow sending flags to the gadgets. It would be useful also other gadgets as well, for instance, to show only UDP or TCP sockets with the `socket-collector` gadget.

mauriciovasquezbernal commented on docs/guides/biolatency.md:
````diff
@@ -0,0 +1,130 @@
+```
+
+The new histogram shows how the number of I/O operations increased
+significantly, passing from 1544 (Normal load) to 936438 (Stressing the I/O).
````
The ones inside the parentheses should be lowercase IIUC from http://englishplus.com/grammar/00000123.htm.

mauriciovasquezbernal reviewed (approved):
LGTM!
